_posts/blog/2018-09-12/2018-09-12-how-to-get-more-buyers-on-openbazaar-with-categories-and-tags.markdown

```diff
@@ -0,0 +1,145 @@
+
+In this example, we enter tags that may match what users enter in Search in the OpenBazaar Discover view.
+
+![Add tags to your OpenBazaar listing](Add tags to your OpenBazaar listing.gif "Add tags to your OpenBazaar listing")
+
+> Tip: Make sure to add your store name as a tag too so when people search for your store name, your listings will show up.
+
+### Step 2: Categories and Filters in Your Store
+
+Once users click on your listing, they arrive in your store. This is where you have an opportunity to hook them and encourage them to shop around more. Your **Store View** will show potential customers all your available listings. 
+
+![OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags](OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png "OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags")
+
+It will make it easier for buyers to shop if you classify your listings. Take time to become familiar with the Store View browsing experience. It will aid you later in listing planning, classification, organization and creation.
+
+#### Classify your Listings for Easy Store Browsing
+
+![Category radio buttons on OpenBazaar](Category radio buttons on OpenBazaar.png "Category radio buttons on OpenBazaar")
+
+A little planning and foresight can make listing creation much easier. Determine which **categories** and possible subcategories you'll support in the beginning. 
+
+In our example store, Variety Bazaar, we have a wide variety of listings. We ensure the potential buyer finds what they want with ease with useful categories.
+
+![Add categories to your OpenBazaar listing](Add categories to your OpenBazaar listing.gif "Add categories to your OpenBazaar listing")
+
+Enter clear, concise and unique category names for your listings. For our example store, we sell clothing, antiques, comic books and fidget spinners. We decided to represent subcategories within clothing by applying more than one category. We also sell retro items within separate categories. These items are all given the extra category 'retro'.
+
+It is important not to confuse tags with categories. Categories are for organizing your store and providing a simple store browsing experience. They are only used for filtering on the store view.
+
+#### Browsing using Listing Filters
+
+When potential buyers browse your store, they will see  a few useful filters to aid them. Understanding these filters will help you plan your store well.
+
+#### Store Search
+
+![Store search bar on OpenBazaar](Store search bar on OpenBazaar.png "Store search bar on OpenBazaar") 
+
+Unlike in the search engines, text entered into this **store search** filters your listings based on their _title_, not their _tags_.
```

Comment:
Again, search does not only use tags.

Maybe "filters your listings based only on their titles."